Week3/BusinessCard/app/src/main/res/anim/bounce.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fillAfter="true"
+    android:interpolator="@android:anim/bounce_interpolator">
+
+    <scale
+        android:duration="500"

```

Comment:
You can also oull this out to dimens, to make it easier to implement multiple similar animations!